x/ecocredit/module/module.go

```diff
@@ -162,7 +162,7 @@ func (a Module) GetTxCmd() *cobra.Command {
 }
 
 // ConsensusVersion implements AppModule/ConsensusVersion.
-func (Module) ConsensusVersion() uint64 { return 2 }
+func (Module) ConsensusVersion() uint64 { return 3 }
```

Comment:
I think we can actually leave this at `2`. We did not use the consensus version in the last upgrade.

https://github.com/regen-network/regen-ledger/blob/v3.0.0/app/stable_appconfig.go#L51-L52